src/o365/graph/commands/groupsetting/groupsetting-set.ts

```diff
@@ -0,0 +1,209 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import * as request from 'request-promise-native';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import Utils from '../../../../Utils';
+import GraphCommand from '../../GraphCommand';
+import { GroupSetting } from './GroupSetting';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id: string;
+}
+
+class GraphGroupSettingSetCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.GROUPSETTING_SET}`;
+  }
+
+  public get description(): string {
+    return 'Updates the particular group setting';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.id = args.options.id;
```

Comment:
ID is a required parameter which is unique for each environment. There is no much point in tracking it